react/types.tsx

```diff
@@ -35,6 +35,7 @@ export interface Field {
    */
   mask?: string
   additionalData?: PostalCodeData | null
+  pattern?: string
```

Comment:
question: why this whole interface isn't imported from `country-data-settings`?

Reply:
because this app (unfortunately) doesn't depend on it :-/

Reply:
I see. But should we create a task about this or something?